mpd/client.go

```diff
@@ -360,6 +360,31 @@ func (c *Client) PlaylistInfo(start, end int) ([]Attrs, error) {
 	return cmd.AttrsList("file")
 }
 
+// Prio set the priority of the specified songs. If end is negative but start
+// is positive, it does it for the song at position start. If both start and
+// end are positive, it does it for positions in range [start, end).
+func (c *Client) Prio(prio, start, end int) error {
```

Comment:
Maybe name this function SetPriority? There is no reason we have to follow MPD's obscure naming.